tinygrad/renderer/assembly.py

```diff
@@ -172,7 +219,7 @@ def cast(a:str, dtype:DType, atype:DType, bitcast=False, u=None, pred=False):
   return lang.render_kernel(kernel, function_name, bufs, c.items())
 
 class PTXLanguage(AssemblyLanguage):
-  kernel_prefix = """.version 7.8
+  kernel_prefix = """.version 7.5
```

Comment:
Why did this change?

Reply:
Couldn't run on my machine with an older cuda install, figured that since we don't use any 7.8 specific features it would be better to change it in the repo to support older architectures as well